### PR TITLE
fix: footer position and missing standard property.

### LINKED ISSUE #1: Anything unique.
Any kind of styling, animations, etc... can be added.

### PR DESCRIPTION
**Pull request by ToluAdegboyega**
Hello! I made a change to center the display and also added the missing standard background-clip property. I have attached before and after images.
 #1 
Screenshot (6756)
Screenshot (6757)

### CODE REVIEW

File: Forms/style.css [hunk 1]
```diff
@@ -46,6 +45,7 @@ h1 {
     background-size: 40%;
     background-position: 50% 50%;
     -webkit-background-clip: text;
+    background-clip: text;
```
Comment (Abhiramborige) [inline Forms/style.css:53]: this has no effect on the style

**Comment by Abhiramborige** [comment]
image

**Comment by Abhiramborige** [comment]
could you please fix this, 
I want a better mobile view

**Comment by Abhiramborige** [comment]
please try to remove the background clip to the emoji (hand wave). 
keep that only to text "Greetings".

**Comment by ToluAdegboyega** [comment]
Done.
Screenshot (6759)

**Comment by Abhiramborige** [comment]
Thanks for the contrib!

**Pull request merged**